database/postgres/save_data.py

```diff
@@ -1,16 +1,31 @@
+from database.models import GamesStatuses
 from database.session import SessionLocal
 from typing import Dict
+from cachetools import TTLCache
 from database.postgres.dto import TeamDTO, CountryDTO, LeagueDTO, GameDTO, SportDTO, PlayerDTO
 from database.postgres.dal import TeamDAL, LeagueDAL, GameDAL, CountryDAL, SportDAL, PlayerDal
 from datetime import datetime
 
+status_cache = TTLCache(maxsize=10, ttl=60*60*24)
```

Comment:
MINOR: If that's 24 hours - too much time.

MINOR: I don't know exactly how this logic is used, since it is part of the scraper, but if there is a possibility of more than one thread running, we should probably make it Thread-Safe (see the sample image below). I am, however, 99% sure that we don't do multiple threads here. I think all of our scrapers are synchronous. 
![image](https://github.com/user-attachments/assets/08e98b37-ed31-4a1c-bad3-e6112ca5115d)